website/docs/r/bigip_ltm_monitor.html.markdown

```diff
@@ -54,3 +54,5 @@ resource "bigip_ltm_monitor" "monitor" {
 * `time_until_up` - (Optional)
 
 * `destination` - (Optional) Specify an alias address for monitoring
+
+* `compatibility` - (Optional) Specifies, when enabled, that the SSL options setting (in OpenSSL) is set to ALL. The default value is enabled."
```

Comment:
Can you specify that it is a boolean, and should be set to either `enabled` or `disabled`?